dockerfiles/stopall.sh

```diff
@@ -50,6 +53,10 @@ if [[ $UNINSTALL_ELK == 1 ]] ; then
   helm uninstall myelastic4hpcclogs
   kubectl get pvc
 fi
+if [[ $UNINSTALL_PROMETHEUS == 1 ]] ; then
```

Comment:
I suspect it may be better to uninstall unconditionally (in the same way as localfile), so you don't need to remember to match the arguments.  (Unless it is common to want prometheus to  remain running).